var/spack/repos/builtin/packages/git/package.py

```diff
@@ -56,7 +55,7 @@ class Git(Package):
     # Also depends_on gettext: apt-get install gettext (Ubuntu)
```

Comment:
Gettext wouldn't install for me, but I overhauled it in #1156 so it should be good now.